apps/api/v1/rest/websoc/src/index.ts

```diff
@@ -0,0 +1,222 @@
+import { LambdaClient } from "@aws-sdk/client-lambda";
+import { PrismaClient } from "@libs/db";
+import { createErrorResult, createOKResult, type InternalHandler } from "ant-stack";
+import type { WebsocAPIResponse } from "peterportal-api-next-types";
+import { ZodError } from "zod";
+
+import {
+  combineResponses,
+  constructPrismaQuery,
+  normalizeQuery,
+  notNull,
+  PeterPortalApiLambdaClient,
+  sortResponse,
+} from "./lib";
+import { QuerySchema } from "./schema";
+
+let prisma: PrismaClient;
+
+const lambda = new LambdaClient({});
```

Comment:
If it's possible to initialize the `LambdaClient` without any meaningful options, then I think the class should provide a default argument.

```suggestion
const lambda = new LambdaClient();
```

Reply:
Yeah unfortunately you have to pass in the empty object. AWS SDK PR when?

Reply:
Oh mb, I thought this was the client that we created lol. I remember one that we used internally for making requests, but I forgot where it was